ansible/templates/minigraph_png.j2

```diff
@@ -191,6 +193,11 @@
         <Hostname>{{ asic }}</Hostname>
         <HwSku>Broadcom-Trident2</HwSku>
       </Device>
+{% endfor %}
+{% for asic in fabric_info %}
+      <Device i:type="Asic">
+        <Hostname>{{ asic['asicname'] }}</Hostname>
+      </Device>
```

Comment:
HwSku required ?

Reply:
I thought the "Device" with type!="Asic" is the one that requires Hwsku.
I can add it to Asic types too if that is required.